acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/proxy/PassThroughAcceptanceTest.java

```diff
@@ -43,6 +43,12 @@ public void ethGetTransactionCountReturnCorrectNumber() {
     assertThat(startTransactionCount.add(BigInteger.ONE)).isEqualTo(endTransactionCount);
   }
 
+  @Test
+  public void mySimpleTest() {
```

Comment:
In the current state this isn't really a test. It lacks an assertion step on the returned value.

Suggestions
- rename the method (let's be honest, `mySimpleTest` isn't the most meaningful name).
- add assert step on returned value (perhaps comparing to the transaction count when directly retrieved from the node?)